tests/rt.conf

```diff
@@ -23,21 +23,21 @@ RUN     | fv3_iau
 RUN     | fv3_lheatstrg                                                                                                           |                                         | fv3 |
 
 # WW3 not working on Cheyenne in the past, need to check if it works now
-COMPILE | SUITES=FV3_GFS_2017,FV3_GFS_2017_gfdlmp WW3=Y                                                                           | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
+COMPILE | APP=ATMW SUITES=FV3_GFS_2017,FV3_GFS_2017_gfdlmp
 RUN     | fv3_gfdlmprad                                                                                                           | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
 RUN     | fv3_gfdlmprad_atmwav                                                                                                    | + wcoss_dell_p3 hera.intel orion.intel  | fv3 |
 RUN     | fv3_wrtGauss_nemsio_c768                                                                                                | +               hera.intel orion.intel  | fv3 |
 
 # Run multigases test in REPRO mode to avoid numerical instability in the deep atmosphere
-COMPILE | SUITES=FV3_GFS_2017_fv3wam 32BIT=Y MULTI_GASES=Y REPRO=Y                                                                |                                         | fv3 |
+COMPILE | APP=ATM SUITES=FV3_GFS_2017_fv3wam 32BIT=Y MULTI_GASES=Y REPRO=Y
```

Comment:
Add ` |                                         | fv3 |`

Reply:
addressed in 277cefb